package.json

```diff
@@ -26,20 +28,20 @@
     "laravel-mix-imagemin": "^1.0.3",
     "node-sass": "^4.14.1",
     "resolve-url-loader": "^3.1.1",
-    "sass-loader": "^8.0.2",
+    "sass-loader": "^9.0.3",
     "vue-template-compiler": "^2.6.11",
     "wp-hookdoc": "^0.2.0"
   },
-  "dependencies": {
-    "@wordpress/element": "^2.16.0",
-    "govuk-frontend": "^3.7.0"
-  },
   "scripts": {
     "start": "wp-scripts start",
     "build": "wp-scripts build && NODE_ENV=production webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "build-comp": "wp-scripts build",
+    "build-comp": "wp-scripts build && npm run production",
     "zip": "zip -r mojblocks.zip ./ -x .git .gitignore *.git/* .DS_Store .package.json .package-lock.json *.scss webpack.config.js mojblocks.zip  postcss.config.css README.md .wordpress-org/**\\* .ds_store vendor/**\\* node_modules/**\\* src/**\\* webpack.config.js n",
     "build:docs": "rm -rf hookdocs/ && jsdoc -c hookdoc-conf.json",
-    "watch": "webpack --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
+    "watch": "webpack --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "production": "NODE_ENV=production webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
```

Comment:
Ooh nice :) when would we want to be running this, just for reference? Would this replace running `npm run build` at the end, before pushing up for a PR?

Reply:
Good spot! I've removed it, thanks.